### DISCUSSION TITLE
digital envelope routines::unsupported

### DISCUSSION THREAD

**Discussion by mtamma** [vercel/community#1622] [Help] [answered]
### Summary

![image](https://user-images.githubusercontent.com/3013625/221357962-2620a40f-bb74-4f9a-8285-8a3b06a812d1.png)

this only happens when the build executes on the vercel. but no errors are shown when I run the build on my locally.

**Comment by wojtekfarmery** [comment, marked as answer]
What is your node js version? v18.12.1 has been known to cause this error. If that's the case I recommend upgrading node js

**Reply by mtamma** [reply]
Thanks for the feedback.
I notice on the vercel we can select the nodejs version of when the deployment process.
thanks, the issue is solved 🙏
